src/components/data-table/_data-table-action.scss

```diff
@@ -136,7 +146,9 @@
   //-------------------------------------------------
   //TOOLBAR BUTTONS
   //-------------------------------------------------
-  .#{$prefix}--overflow-menu.#{$prefix}--toolbar-action {
+  .#{$prefix}--overflow-menu.#{$prefix}--toolbar-action,
+  // DEPRECATED prefer .#{$prefix}--overflow-menu.#{$prefix}--toolbar-action instead
```

Comment:
I would argue this shouldn't be a deprecation item at all ... toolbars can totally have non-overflow-menu items, so why would we require `.bx--overflow-menu` on all of them?

Reply:
This is coming from a design change in `v10`. Data table toolbar actions are now in an overflow menu instead of inline like in `v9`. cc: @designertyler will data table be supporting inline actions in the toolbar as an option instead of having them in an overflow menu?

Reply:
Ah okay, that makes sense ... We can leave it as is then :+1: